duck 4.6.5

dkocher opened the pull request.

mikemcquaid commented on Library/Formula/duck.rb:
```diff
@@ -1,16 +1,9 @@
-    sha1 "05353941783e34f1931b0cdd78a89f245d8daf33" => :yosemite
-    sha1 "a2120b7b4b7ae15f9b941c24bb9d4f7c36392e74" => :mavericks
-    sha1 "fb6e8e20d702cf3d9808e758328cd7ebc8830283" => :mountain_lion
-  end
```
Just leave the bottle block as-is and we'll update it when we pull, thanks!

mikemcquaid commented:
Still needs rebased, it seems. Rebase and squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master and then mark each commit other than the first as squash or fixup then git push -f. For future reference you can do this in the existing pull request and it'll be automatically updated (but don't worry about it for this time). Thanks!

dkocher commented:
We have now modified our continuous build to pull changes from upstream before applying the changes. Also, the bottles will no longer be removed. As we have deleted and created a new branch this pull request got closed.

dkocher commented:
Now in #37183.
